vars/runPipeline.groovy

```diff
@@ -206,6 +209,12 @@ def testDownstreamProject (name) {
                     sh 'dub test --compiler=$DC'
                     break;
 
+                case 'sociomantic-tsunami/ocean':
+                    sh 'git submodule update --init'
+                    sh 'make d2conv' // requires d1to2fix installed
+                    sh 'make test DVER=2 F=production'
+                    break;
```

Comment:
Can we just download and make `d1to2fix` here then for the time being ?

Reply:
Yes, I plan to download d1to2fix and libebtree debian packages from our bintray repos and extract required binaries. Just haven't got to update this PR yet.